etna/analysis/outliers/isolation_forest_outliers.py

```diff
@@ -35,7 +35,9 @@ def _select_features(
             "Changing the defaults there should be exactly one option set: features_to_use or features_to_ignore"
         )
     features_to_ignore = list(set(features_to_ignore) - {in_column})
-    df = df.drop(columns=features_to_ignore, level="feature")
+    if len(features_to_ignore) > 0:
```

Comment:
Let's add which versions of pandas fail here. We added such comments in other similar places before.